Base/usr/share/man/man8/sysctl.md

```diff
@@ -0,0 +1,51 @@
+## Name
+
+sysctl - configure kernel parameters at runtime
+
+## Synopsis
+
+```**sh
+# sysctl [-a] [variable[=value]]
+```
+
+## Description
+
+sysctl is a utility for managing kernel configuration paramaters at runtime.
+This requires root privileges, and can crash your system.
+Available parameters are listed under /proc/sys/.
+
+## Options
+
+* `-a`: Display all kernel parameters and associated values
+
+## Arguments
+
+* `variable`: Retrieve the specified parameter
+* `variable=value`: Set the specified parameter to the specified value
+
+## Files
+
+* `/proc/sys` - source of kernel parameters
+
+## Examples
+
+View all parameters:
+
+```sh
+# sysctl -a
+```
+
+View `ubsan_is_deadly` parameter:
+
+```sh
+# sysctl ubsan_is_deadly
+ubsan_is_deadly = 1
+```
+
+Set `ubsan_is_deadly` paramater to zero (disabled):
+
+```sh
+# sysctl ubsan_is_deadly=0
+ubsan_is_deadly = 1 -> 0
+```
+
```

Comment:
Please remove this black line at eof or it will cause problems after #5687 gets merged.